src/Controls/src/Core/HandlerImpl/VisualElement.Impl.cs

```diff
@@ -56,7 +56,20 @@ protected virtual void ArrangeOverride(Rectangle bounds)
 
 			var newRect = this.ComputeFrame(bounds);
 
-			Layout(newRect);
+			Bounds = newRect;
+			Handler?.SetFrame(Bounds);
+		}
+
+		// TODO: MAUI
+		// This is here to support layout calls from legacy code
+		// This should go away once we get everything piping through
+		// Maui based layout code
+		public void Layout(Rectangle bounds)
```

Comment:
I moved this code here so that it's closer to the Arrange call that will eventually replace it